specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -8024,7 +8123,7 @@
       },
       "type": "object"
     },
-    "ToggleSettings": {
```

Comment:
>ToggleSettings [](start = 5, length = 14)

Again these are breaking changes but as per the author this types are not being used; based on that I'm allowing this change.